script.js

```diff
@@ -1,4 +1,5 @@
 let button = document.getElementById('js-calc-cipher');
+let userNameArea = document.getElementById('user-name');
```

Comment:
ik ga het zo laten, maar eigenlijk zou het beter zijn om dat allemaal in een `<form>` te steken en dat te submitten (dan kunt ge uw form laten serialiseren in 1 keer naar JSON)

Reply:
deze ga ik sws voor u laten dan, want dat gaat voor mij nog redelijk zoeken zijn 😅